packages/ui/src/components/molecules/Table/styles.scss

```diff
@@ -0,0 +1,118 @@
+[data-fs-table] {
+  // --------------------------------------------------------
+  // Design Tokens for Table
+  // --------------------------------------------------------
+
+  // Default properties
+
+  // Cell
+  --fs-table-cell-padding-x             : var(--fs-spacing-3);
+  --fs-table-cell-padding-y             : var(--fs-spacing-1);
+
+  // Head
+  --fs-table-head-weight                : var(--fs-text-weight-bold);
+  --fs-table-head-bkg-color             : none;
+
+  // Footer
+  --fs-table-footer-weight              : var(--fs-table-head-weight);
+  --fs-table-footer-bkg-color           : none;
+
+  // Colored
+  --fs-table-colored-bkg-color          : var(--fs-color-neutral-1);
+  --fs-table-colored-border-radius      : var(--fs-border-radius);
+
+  // Bordered
+  --fs-table-bordered-border-width      : var(--fs-border-width);
+  --fs-table-bordered-border-color      : var(--fs-border-color-light);
+
+
+  // --------------------------------------------------------
+  // Structural Styles
+  // --------------------------------------------------------
+
+  width: 100%;
+  display: block;
+  overflow-x: auto;
+  white-space: nowrap;
+
+  th, td {
```

Comment:
Why are we using html tags instead of data attributes? Is not it better to use data attributes to avoid adding styles in regular html tags?